packages/forklift-console-plugin/src/modules/Providers/views/migrate/ProvidersCreateVmMigrationContext.tsx

```diff
@@ -0,0 +1,48 @@
+import { createContext, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { produce } from 'immer';
+
+import { V1beta1Provider } from '@kubev2v/types';
+
+export interface CreateVmMigrationContextData {
+  selectedIds: string[];
```

Comment:
selected ids are actually the `vms` field in `V1beta1Provider`
having 2 fields with the same data may cause inconsistencies, do you use the `selectedIds` for something else ?  for can you use the `vms` field in `V1beta1Provider` to store the `vms` ?

Reply:
Originally the plan was to pass just the IDs of the selected VMs. However they are not very user friendly so I'll switch to full VM objects. This should also help with displaying the table with VMs in the model (as requested).

Reply:
:+1: nice !

Reply:
note: we may have lots of selected vms, for efficiency it may be an option to not store the vms, and re-fetch them when user open the modal, WDYT ?